docs/quickstart.rst

```diff
@@ -55,8 +55,7 @@ There's a lot going on here, so let's walk you through it step by step.
    sure that we ignore messages from ourselves. We do this by checking if the :attr:`Message.author`
    is the same as the :attr:`Client.user`.
 5. Afterwards, we check if the :class:`Message.content` starts with ``'$hello'``. If it does,
-   then we send a message in the channel it was used in with ``'Hello!'``. This is a basic way of 
-   handling commands, which can be later automated with the :doc:`./ext/commands/index` framework.
+   then we send a message in the channel it was used in with ``'Hello!'``.
```

Comment:
Why did you remove stuff from this?

Reply:
Removed it in order to not direct users towards ext.commands when there is a slash example 2 lines below it.

Reply:
But ext.commands work fully well with slash commands

Reply:
It does, but isn't what I would consider the logical next step after completing that example, as this text would imply.

Reply:
I mean prefixed commands have there upside and iirc discord.Bot is still missing some ext.commands features, I am also planning on deprecating discord.Bot soon and making prefix optional in commands.Bot

Reply:
At this point we are getting into library design talk, and as I cannot really see into the future of how the library will shape up, I don't think I can just the example to that as of now. All of this can be adjusted as the library further develops :)

Reply:
Ye I just think this shouldn’t be changed but I would like others opinion

Reply:
Please revert this change